tachyon/zk/plonk/halo2/synthesizer.h

```diff
@@ -32,7 +32,7 @@ class Synthesizer {
       advice_columns_vec_[i] =
           std::vector<Evals>(constraint_system->num_advice_columns());
       advice_blinds_vec_[i] =
-          std::vector<F>(constraint_system->num_advice_columns());
+          std::vector<F>(constraint_system->num_advice_columns(), F::Zero());
```

Comment:
It seems like we don't need to initialize the values, since `SetAdviceColumn()` will be filled with the values. But just add a comment that why this is safe.